packages/webui/src/styles/base.css

```diff
@@ -5,7 +5,7 @@
 @tailwind utilities;
 @tailwind variants;
 
-html {
+body {
```

Comment:
Switched to `body` as applying this to the `html` was causing the REM-based spacing scale to differ from what we have in designs.

Reply:
Fair enough.  Not sure I can remember why I did this on `html` in the first place, but just out of curiosity can you share an example of what wasn't working correctly with this, and why the change to `body` fixes it... I feel like there's a lesson for me to learn here.

Reply:
For sure, sorry my comment was a bit vague.

So Tailwind's spacing values use rem units. Rem values are relative to the font-size of the "root" (`<html />`) element, so if the font-size on the `html` element is `16px`, `0.5rem` will compute to `8px`. If the `html` element's font-size is changed to `14px`, `0.5rem` will compute to `7px`.

There are probably some articles that explain it better than I can, but here is a visual example of the impact it has on the computed values:

**Before - 14px font-size on the `html` element**
<img width="1425" alt="Screenshot 2023-11-06 at 09 52 53" src="https://github.com/FormidableLabs/envy/assets/9557798/69ff2c87-b444-4311-b2d3-7285cb6bfc21">

**After - 14px font-size on the `body` element**
<img width="1425" alt="Screenshot 2023-11-06 at 09 52 46" src="https://github.com/FormidableLabs/envy/assets/9557798/51235480-a564-4edd-bb9a-dc69220f8172">

font-size on the `body` element, however, does not impact how `rem` units are computed which means we get the desired base font size, and do not affect the spacing scale.

Reply:
I see it too.  @KenanYusuf do we need to make any other adjustments to retain the previous font size?